backend/app/services/price_fetcher_integration.py

```
@@ -0,0 +1,375 @@
+                        try:
+                            eur_rate = await self.yahoo_fetcher.fetch_fx_rate("USD", "EUR")
+                            if eur_rate:
+                                price_eur = price_usd * eur_rate
+                                result["current_price"] = price_eur
+                                result["currency"] = "EUR"
+                            else:
+                                # Fallback conversion
+                                price_eur = price_usd * Decimal("0.92")
+                                result["current_price"] = price_eur
+                                result["currency"] = "EUR"
+                        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)